yt/geometry/oct_container.pyx

```diff
@@ -217,7 +217,7 @@ cdef class OctreeContainer:
             ind32[i] = ind[i]
         self.get_root(ind32, &next)
         # We want to stop recursing when there's nowhere else to go
-        while next != NULL and level <= max_level:
+        while next != NULL and level < max_level:
```

Comment:
this seems like a substantive change -- is it fixing an error?

Reply:
Imagine that the `get` function is called with `max_level = 0`, then we'd expect one and only one iteration to be done (as `level` starts with a value of `-1`). Prior to the change, up to two iterations would be performed, eventually returning an oct at `max_level + 1`.

This never triggered an error as the `max_level` optional argument was not used anywhere.